src/Polytopes/PolyhedralFan/properties.jl

```diff
@@ -22,23 +22,64 @@ Base.length(iter::PolyhedralFanRayIterator) = nrays(iter.fan)
 """
     rays(PF::PolyhedralFan)
 
-Returns the rays of a polyhedral fan.
+Return the rays of `PF`.
+
+# Examples
+The rays of a normal fan of a cube point in every positive and negative unit
+direction.
+```jldoctest
+julia> C = cube(3);
+
+julia> NF = normal_fan(C);
+
+julia> collect(rays(NF))
+6-element Vector{Polymake.Vector{Polymake.Rational}}:
+ pm::Vector<pm::Rational>
+1 0 0
+ pm::Vector<pm::Rational>
+-1 0 0
+ pm::Vector<pm::Rational>
+0 1 0
+ pm::Vector<pm::Rational>
+0 -1 0
+ pm::Vector<pm::Rational>
+0 0 1
+ pm::Vector<pm::Rational>
+0 0 -1
+```
 """
 rays(PF::PolyhedralFan) = PolyhedralFanRayIterator(PF)
 
 
 """
     maximal_cones(PF::PolyhedralFan)
 
-Returns the maximal cones of a polyhedral fan.
+Return the maximal cones of `PF`.
+
+# Examples
+Here we ask for the the number of rays for each maximal cone of the face fan of
+the 3-cube and use that `maximal_cones` returns an iterator.
+```jldoctest
+julia> PF = face_fan(cube(3));
+
+julia> for c in maximal_cones(PF)
+       println(nrays(c))
+       end
+4
+4
+4
+4
+4
+4
```

Comment:
Maybe something likethis would be more Julia-ish?
```suggestion
julia> map(nrays, maximal_cones(PF))
6-element Vector{Int64}:
 4
 4
 4
 4
 4
 4
```